build/getFolderNames.js

```diff
@@ -0,0 +1,10 @@
+const { lstatSync, readdirSync } = require('fs');
+const { join } = require('path');
+
+export const getSubDirectoryNames = (root) => {
+  return readdirSync(root)
+    .filter(name => {
+      let fullPath = join(root, name);
```

Comment:
I would use `const` if you aren't changing anything

Reply:
nice